.github/workflows/build.yaml

```diff
@@ -88,8 +88,10 @@ jobs:
       date: ${{ inputs.date }}
       package-name: libcuspatial
       package-type: cpp
+      # build for every combination of arch and CUDA version, but only for the latest Python
+      matrix_filter: group_by([.ARCH, (.CUDA_VER|split(".")|map(tonumber)|.[0])]) | map(max_by(.PY_VER|split(".")|map(tonumber)))
```

Comment:
```suggestion
```

This was a bad read of the file... we don't need a filter like this for `wheels-publish`.